cmd/bridge/config/auth/authoptions.go

```diff
@@ -234,7 +233,7 @@ func (c *completedOptions) getAuthenticator(
 		authLoginSuccessEndpoint = proxy.SingleJoiningSlash(baseURL.String(), server.AuthLoginSuccessEndpoint)
 		oidcClientSecret         = c.ClientSecret
 		// Abstraction leak required by NewAuthenticator. We only want the browser to send the auth token for paths starting with basePath/api.
-		cookiePath       = proxy.SingleJoiningSlash(baseURL.Path, "/api/")
+		cookiePath       = proxy.SingleJoiningSlash(baseURL.Path, "/")
```

Comment:
this should be reverted/fixed, the logout code will need to access the session cookies but the cookie should actually rather stay within the /api path